src/compiler/scala/reflect/quasiquotes/Parsers.scala

```diff
@@ -57,8 +57,8 @@ trait Parsers { self: Quasiquotes =>
       override implicit lazy val fresh: FreshNameCreator = new FreshNameCreator(nme.QUASIQUOTE_PREFIX)
 
       // Do not check for tuple arity. The placeholders can support arbitrary tuple sizes.
-      override def makeSafeTupleTerm(trees: List[Tree], offset: Offset): Tree = treeBuilder.makeTupleTerm(trees)
-      override def makeSafeTupleType(trees: List[Tree], offset: Offset): Tree = treeBuilder.makeTupleType(trees)
+      override def makeSafeTupleTerm(trees: List[Tree]): Tree = treeBuilder.makeTupleTerm(trees)
+      override def makeSafeTupleType(trees: List[Tree]): Tree = treeBuilder.makeTupleType(trees)
```

Comment:
I don't know enough about quasiquotes to understand if these overrides should also include `makeSafeFunctionType`. Maybe someone else does?

Reply:
Yeah, I meant to ask about this. Would be good if someone could comment one way or the other.